index.html

```diff
@@ -21,6 +21,10 @@
   <body>
     <!-- start of navbar -->
     <header>
+    
+      <div class="Brand-logo">
+        <a href="#"></a>
+      </div>
```

Comment:
instead of this line 
add this: <a class="navbar-brand" href="#">Navbar</a>

the styling is already done for you here ^